pipeline/src/pipeline.rs

```diff
@@ -920,6 +915,10 @@ impl<T: FieldElement> Pipeline<T> {
         self.output_dir.as_ref().map(|p| p.as_ref())
     }
 
+    pub fn force_overwrite(&self) -> bool {
```

Comment:
this sounds like you're telling pipeline to force overwrite things. Maybe `is_force_overwrite`?

Reply:
fixed